javascript/javascript3/week1/weatherapp/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@100&display=swap" rel="stylesheet"> <!--font google-->
+    <script src="https://kit.fontawesome.com/f44cd04f18.js" crossorigin="anonymous"></script>  <!--font awesome-->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css" integrity="sha256-kLaT2GOSpHechhsozzB+flnD+zUyjE2LlfWPgU04xyI=" crossorigin=""/>
+    <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js" integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM=" crossorigin=""></script>
+    <title>Weather App</title>
+</head>
+<body>
+
+    <div id="container" class="container">       
+        <div class="main" id="main">        
+            <div class="input">
+                <div class="input-wrapper">
+                    <label for="input"></label>
+                    <input id="input" placeholder="Search for a city...">
+                    <button id="weatherBtn" class="weather-btn">Search</button>
```

Comment:
It is recommended to keep only one naming convention. Either camelCase or lowercase-with-hyphens.